### PR TITLE
A beter lore for changling

### PR DESCRIPTION
**Pull request by Supermichael777**
I feel this is better.

### CODE REVIEW

**Comment by Aloraydrel** [comment]
Did you make sure this is canon?

**Comment by Armhulen** [comment]
i support this

**Review by Anonmare — COMMENTED** [review]
>By the time this was discovere 

Slight typo here. Missing a "d".

File: code/game/gamemodes/intercept_report.dm [hunk 1]
```diff
@@ -12,10 +12,10 @@
 			CMP subsequently gained. Be vigilant for any large organisms rapidly spreading across the station, as they are classified as a level 5 biohazard and critically dangerous. Note that \
 			this organism seems to be weak to extreme heat; concentrated fire (such as welding tools and lasers) will be effective against it."
 		if("changeling")
-			text += "The Gorlex Marauders have announced the successful raid and destruction of Central Command containment ship #S-[rand(1111, 9999)]. This ship housed only a single prisoner - \
-			codenamed \"Thing\", and it was highly adaptive and extremely dangerous. We have reason to believe that the Thing has allied with the Syndicate, and you should note that likelihood \
-			of the Thing being sent to a station in this sector is highly likely. It may be in the guise of any crew member. Trust nobody - suspect everybody. Do not announce this to the crew, \
-			as paranoia may spread and inhibit workplace efficiency."
+			text += ""A follow up on the investigation of a derelict ship from the age of Orion found unreported corpses. Though they were dressed in the crew's clothing; carbon dating and bone-structure \
```
Comment (Cyberboss) [inline code/game/gamemodes/intercept_report.dm]: Dat double quote

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/intercept_report.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/gamemodes/intercept_report.dm]: fug

**Comment by Xhuis** [comment]
y tho

**Comment by Armhulen** [comment]
are you joking just fix the two problems

**Pull request closed without merging**